lib/scsi_defs.h

```diff
@@ -80,5 +97,282 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  // SCSI Reference Manual Table 95 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  struct Read6 {
+    OpCode op_code = OpCode::kRead6;
+    uint8_t reserved: 3;
+    uint32_t logical_block_address: 21;
+    uint8_t transfer_length: 8;
+    ControlByte control_byte;
+  } ABSL_ATTRIBUTE_PACKED;
+  static_assert(sizeof(Read6) == 6);
+  
+  // SCSI Reference Manual Table 97 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf  
+  struct Read10 {
+    OpCode op_code = OpCode::kRead10;
+    uint8_t rd_protect: 3;
+    bool dpo: 1;
```

Comment:
Similar to https://github.com/googleinterns/scsi2nvme/pull/27#discussion_r430594682, should these have comments?

Reply:
Good catch. Will add comments!